app/models/api/hses.rb

```diff
@@ -55,7 +55,8 @@ class Api::Hses::Issues < ApiRequest
 
   def initialize(user:, params: {})
     @username = user["uid"]
-    @page = params[:page] || 1
+    @page = params.fetch(:page, 1)
```

Comment:
[praise] I've never used this before but now I'm wondering how I made it so many years without realizing this is a thing??